dev_apps/music-nga/test/marionette/lib/music.js

```diff
@@ -0,0 +1,493 @@
+/* global require, module */
+'use strict';
+
+var assert = require('assert');
+
+function Music(client, origin) {
+  this.client = client;
+  this.origin = origin || ('app://' + Music.DEFAULT_ORIGIN);
+  this.actions = client.loader.getActions();
+}
+
+module.exports = Music;
+
+Music.DEFAULT_ORIGIN = 'music-nga.gaiamobile.org';
+
+Music.Selector = Object.freeze({
+  viewFrame: '#view-stack iframe',
+  secondaryViewFrame: '#view-stack iframe:nth-child(2)',
+  homeViewFrame: 'iframe[src*="views/home/index.html"]',
```

Comment:
You can also do something like:

```
'iframe[data-view-id="home"]'
```

It might be a little shorter and maybe less prone to breaking if the paths would happen to change in the future.